js/util.js

```diff
@@ -37,6 +37,13 @@
       var cases = [2, 0, 1, 1, 1, 2];
 
       return titles[(number % 100 > 4 && number % 100 < 20) ? 2 : cases[(number % 10 < 5) ? number % 10 : 5]];
+    },
+    toggleClassHidden: function (elem, isShow) {
+      if (isShow) {
+        elem.classList.add('hidden');
+      } else {
+        elem.classList.remove('hidden');
+      }
```

Comment:
По ТЗ мы не должны поддерживать IE11, а значит можем обойтись без этой вспомогательной функции и использовать [второй аргумент в методе `classList`](https://caniuse.com/#search=classlist%20toggle) : `classList.toggle('hidden', true)`.

А вообще тут можно было сделать тернарник :
```suggestion
    toggleClassHidden: function (elem, isShow) {
      elem.classList[isShow ? 'add' : 'remove']('hidden');
    }
```